src/views/Dashboard/Customer/Home/GeneticData/RequestAnalysis/AnalystDetail.vue

```diff
@@ -75,12 +80,25 @@ export default {
     Button
   },
 
+  data: () => ({
+    price: null,
+    profileImage: null
+  }),
+
   props: {
     show: Boolean,
     service: Object,
     experiences: Array
   },
 
+  mounted() {
+    this.price = `${this.formatBalance(this.service.priceDetail[0].totalPrice)} ${this.service.priceDetail[0].currency}`
+    
+    if (!this.service.analystsInfo.info.profileImage) {
+      this.profileImage = "https://ipfs.debio.network/ipfs/QmQ9Zvn4pMNMr3nVTGzYE9zGQfyFrhyAHWZNjm76pJmvRn"
```

Comment:
The DeBio logo shouldn't be hardcoded like this. Wouldn't it be better to turn it into an Icon?.

Please ask @bukandicki for help. @bukandicki, please help @ciniiia with this.